std_msgs/Header header
float64 roll
float64 pitch
float64 yaw
float64 gyro_X
float64 gyro_Y
float64 gyro_Z
float64 accel_X
float64 accel_Y
float64 accel_Z
float64 latitude
float64 longitude
float64 altitude
float64 east_Vel
float64 north_Vel
float64 celestial_Vel
float64 gps_Time


